tests/test_Voting.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+"""
+Testing KoalaBot VoteCog
+
+Commented using reStructuredText (reST)
+"""
+# Futures
+
+# Built-in/Generic Imports
+import asyncio
+# Libs
+from unittest import TestCase
+
+import discord
+import discord.ext.test as dpytest
+import mock
+import pytest
+from discord.ext import commands
+
+# Own modules
+import KoalaBot
+from cogs import Voting
+from utils import KoalaDBManager
+
+
+class Fake:
+    def __getattr__(self, item):
+        setattr(self, item, Fake())
+        return getattr(self, item)
+
+
+ctx = Fake()
+ctx.author.id = 1234
+ctx.guild.id = 4567
+cog = None
+db_manager = KoalaDBManager.KoalaDBManager("votingTest.db", KoalaBot.DB_KEY)
+db_manager.create_base_tables()
+
+
+def setup_function():
+    """ setup any state specific to the execution of the given module."""
+    global cog
+    bot = commands.Bot(command_prefix=KoalaBot.COMMAND_PREFIX)
+    cog = Voting.Voting(bot)
+    bot.add_cog(cog)
+    dpytest.configure(bot)
+
+
+@pytest.fixture(scope='session', autouse=True)
+def setup_is_dpytest():
+    KoalaBot.is_dpytest = True
+    yield
+    KoalaBot.is_dpytest = False
+
+
+@pytest.mark.asyncio
+async def test_vote():
```

Comment:
Each test should be short, with in most cases only 1 assert/ verify_message. Longer tests like these become difficult to troubleshoot . Having this in addition to each of these being called individually is also allowed.

each test should also have a very descriptive name as this is what will be used to find what is wrong, feel free to ask if help with the naming is required.

In my cases I would have tests as such
function before_each_test():
    reset dpytest

function test_create():
    test k!vote create x

and then call it again after wiping

function test_addRole():
    test_create()
    test k!vote addRole x